brayns/common/geometry/Cone.h

```diff
@@ -18,38 +18,26 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 
-#ifndef CONE_H
-#define CONE_H
+#pragma once
 
-#include <brayns/common/types.h>
+#include "ExtendedCone.h"
 
 namespace brayns
 {
-struct Cone
+struct Cone : ExtendedCone
 {
-    Cone(const Vector3f c, const Vector3f u, const float cr, const float ur,
-         const float ts = 0.f, const Vector2f t = Vector2f(0.f, 0.f))
-        : center{c}
-        , up{u}
-        , centerRadius{cr}
-        , upRadius{ur}
-        , timestamp{ts}
-        , texture_coords{t}
-        , materialID(-1)
+    Cone(const Vector3f c = {0.f, 0.f, 0.f}, const Vector3f u = {0.f, 0.f, 0.f},
+         const float cr = 0.f, const float ur = 0.f, const float ts = 0.f,
+         const Vector2f t = Vector2f(0.f, 0.f), int32 m = -1)
```

Comment:
Thanks to  aggregate initialization this constructor is not needed in C++11 except for providing default values (which you could do in the struct declaration, but I guess it wouldn't work for ISPC). Consider removing it.

If you remove it, I would rename ExtendedFoo to Foo. I don't see why we need Extended if there isn't a basic counterpart.